pattoo_shared/installation/packages.py

```diff
@@ -24,6 +24,29 @@ def get_package_version(package_name):
     return version[1]
 
 
+def check_outdated_packages(packages, verbose=False):
+    """Check for outdated packages and reinstall them.
+
+    Args:
+        packages: A list of pip packages parsed from the requirement file
+
+    Returns:
+        None
+
+    """
+    # Say what we are doing
+    if verbose:
+        print('Checking for outdated packages')
+    for package in packages:
+        # Get packages with versions from pip_requirements.txt
+        requirement_package = package.split('==', 1)
```

Comment:
How do you handle `>=` if the installed package version is greater what `pattoo` desires?

Installed package is `v100` and `pattoo` requires `v89`, does `v100` get uninstalled? That would be undesirable. You'll need to add this to your unittesting too.

Reply:
Okay, I'm going to make modifications for handling >= now.
So the later versions shouldn't get uninstalled/replaced if a lower version is specified in the requirements file?
I think this might cause some conflicts but I'm going to do some testing to confirm

Reply:
- Or at least give the user the option to consider whether they want to overwrite. Then again, is it all that important if it is going to be running in a dedicated `venv` all the time?
- What guidance would you give to future developers so that they can install the packages and be able to run the scripts in the `bin/` directory without issue? That will need to be documented.